onyo/lib/onyo.py

```diff
@@ -4,8 +4,9 @@
 import shutil
 import string
 import subprocess
+import os
```

Comment:
I also want to mention, that we don't use os anywhere else in onyo.py otherwise (which we did in the beginning, but then decided to remove it), because for most use cases pathlib provides the same functionality, especially for things like os.walk, and it helps just having one path-handling library in usage!

Reply:
Yes and upon further reflection, `pathlib` should be used over `os` unless there's a very good reason to use `os`. In this case the point is moot, since the code using `os.walk` has been removed in favor of `self.assets`  with some tweaking of the output.